corehq/apps/locations/expressions.py

```diff
@@ -0,0 +1,20 @@
+from corehq.apps.userreports.specs import TypeProperty
+from dimagi.ext.jsonobject import JsonObject
+
+
+class LocationTypeSepc(JsonObject):
+    type = TypeProperty('location_type_name')
+
+    def __call__(self, item, context=None):
+        try:
+            doc_id = item['_id']
+            from corehq.apps.locations.models import SQLLocation
+            sql_location = SQLLocation.objects.filter(location_id=doc_id)
```

Comment:
Is it right we want to add cache because this will called as many times per location as the number of times the expression is used. I will cache it
There is no other way that this gets called multiple times per location, right?